Xamarin.RevenueCatUI.iOS/clear.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+rm -rf Output
+rm -rf Carthage
```

Comment:
i am confused. are we using Carthage here?

Reply:
yeah we are not using carthage. removed